index.html

```diff
@@ -244,6 +244,20 @@ <h1>I'm Naan.
                     </div>
                 </div>
             </div>
+            <section class="contact_me">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+                <form action="https://formspree.io/f/xvonabyg"
+                method="POST">
+                    <ul>
+                        <li><input value="" placeholder="Enter your name" id="name" type="text" name="user_name" maxlength="30" required></li>
+                        <li><input value="" placeholder="Enter your email" id="email" type="email" name="email" required></li>
+                    </ul>
+                    <textarea value="" id="message" placeholder="Write your message here" type="text" name="message" maxlength="500" required></textarea>
+                    <button type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly make sure that the `hover`, `enabled`, `disabled`, and `click` properties for the button match that of the Figma design, to enable you to have a pixel-perfect design, which is exactly what you'll be doing in your nearest job 😉.


- Your design
<img width="74" alt="wrong" src="https://user-images.githubusercontent.com/41163571/220788862-97edf7a6-78cf-4bcd-b14d-fddff1a10c65.png">

- Figma design
<img width="400" alt="right" src="https://user-images.githubusercontent.com/41163571/220789030-3bdfcabd-c2df-4e61-9550-b11c1fbdcaaa.png">